### PR TITLE
Update CacheExtensions.xml

### LINKED ISSUE #6611: please add description for IMemoryCache Interface method
hi 
Please add description for  [IMemoryCache Interface](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.caching.memory.imemorycache?view=dotnet-plat-ext-5.0) method  
like set ?

### PR DESCRIPTION
**Pull request by Naidile-P-N**
Adding docs for Set method with absolute expiration

Fixes #6611

### CODE REVIEW

**Review by tarekgh — APPROVED** [review]

**Comment by Naidile-P-N** [comment]
@tarekgh Thanks! My first PR ! Excited to get it merged

**Comment by tarekgh** [comment]
@Naidile-P-N thanks for submitting the PR. we are looking forward seeing more from you :-)

@gewarren do you mind having a quick look at this one and merge it if it is ok? Thanks!

File: xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml [hunk 1]
```diff
@@ -348,13 +348,13 @@
...
-        <param name="absoluteExpiration">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TItem">The generic type for the value stored.</typeparam>
+        <param name="cache">Represents a local in-memory cache whose values are not serialized.</param>
+        <param name="key">An object identifying the requested entry.</param>
+        <param name="value">The located value or null.</param>
+        <param name="absoluteExpiration">A <see cref="T:System.DateTimeOffset" /> representing the expiration time in absolute terms.</param>
+        <summary>Creates or overwrites the entry for the key in cache and sets value with an absolute expiration date.</summary>
+        <returns>Returns the set value.</returns>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml:357]:
```suggestion
        <returns>The value that was set.</returns>
```

File: xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml [hunk 2]
```diff
@@ -348,13 +348,13 @@
...
-        <param name="value">To be added.</param>
-        <param name="absoluteExpiration">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TItem">The generic type for the value stored.</typeparam>
+        <param name="cache">Represents a local in-memory cache whose values are not serialized.</param>
+        <param name="key">An object identifying the requested entry.</param>
+        <param name="value">The located value or null.</param>
+        <param name="absoluteExpiration">A <see cref="T:System.DateTimeOffset" /> representing the expiration time in absolute terms.</param>
+        <summary>Creates or overwrites the entry for the key in cache and sets value with an absolute expiration date.</summary>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml:356]:
```suggestion
        <summary>Creates or overwrites the specified entry in the cache and sets the value with an absolute expiration date.</summary>
```

File: xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml [hunk 3]
```diff
@@ -348,13 +348,13 @@
...
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="absoluteExpiration">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TItem">The generic type for the value stored.</typeparam>
+        <param name="cache">Represents a local in-memory cache whose values are not serialized.</param>
+        <param name="key">An object identifying the requested entry.</param>
+        <param name="value">The located value or null.</param>
+        <param name="absoluteExpiration">A <see cref="T:System.DateTimeOffset" /> representing the expiration time in absolute terms.</param>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml:355]:
```suggestion
        <param name="absoluteExpiration">The expiration time in absolute terms.</param>
```

File: xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml [hunk 4]
```diff
@@ -348,13 +348,13 @@
...
-        <param name="cache">To be added.</param>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="absoluteExpiration">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TItem">The generic type for the value stored.</typeparam>
+        <param name="cache">Represents a local in-memory cache whose values are not serialized.</param>
+        <param name="key">An object identifying the requested entry.</param>
+        <param name="value">The located value or null.</param>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml:354]:
```suggestion
        <param name="value">The located value or <see langword="null" />.</param>
```

File: xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml [hunk 5]
```diff
@@ -348,13 +348,13 @@
...
-        <typeparam name="TItem">To be added.</typeparam>
-        <param name="cache">To be added.</param>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="absoluteExpiration">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TItem">The generic type for the value stored.</typeparam>
+        <param name="cache">Represents a local in-memory cache whose values are not serialized.</param>
+        <param name="key">An object identifying the requested entry.</param>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml:353]:
```suggestion
        <param name="key">The entry to create or overwrite.</param>
```

File: xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml [hunk 6]
```diff
@@ -348,13 +348,13 @@
...
       <Docs>
-        <typeparam name="TItem">To be added.</typeparam>
-        <param name="cache">To be added.</param>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="absoluteExpiration">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TItem">The generic type for the value stored.</typeparam>
+        <param name="cache">Represents a local in-memory cache whose values are not serialized.</param>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml:352]:
```suggestion
        <param name="cache">A local, in-memory cache whose values are not serialized.</param>
```

File: xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml [hunk 7]
```diff
@@ -348,13 +348,13 @@
...
       </Parameters>
       <Docs>
-        <typeparam name="TItem">To be added.</typeparam>
-        <param name="cache">To be added.</param>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="absoluteExpiration">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="TItem">The generic type for the value stored.</typeparam>
```
Comment (gewarren) [inline xml/Microsoft.Extensions.Caching.Memory/CacheExtensions.xml:351]:
```suggestion
        <typeparam name="TItem">The type of the value.</typeparam>
```

**Review by gewarren — COMMENTED** [review]
Thank you @Naidile-P-N . I left a few suggestions to standardize how parameters are described. You can see the full wording guidelines here: https://github.com/dotnet/dotnet-api-docs/wiki.

**Comment by Naidile-P-N** [comment]
@gewarren Thanks. I have committed the suggested changes, can you please review.

**Review by gewarren — APPROVED** [review]
LGTM. Can you sign the license so we can merge it?

**Comment by Naidile-P-N** [comment]
@gewarren I have signed the licence.

**Pull request merged**